test_client_context.py

```diff
@@ -0,0 +1,30 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from client_context import ClientContext
+from fh_state_base_holder import FeatureStateBaseHolder
+
+
+class TestClientContext(TestCase):
+
+    @patch('featurehub_repository.FeatureHubRepository')
+    @patch('edge_service.EdgeService')
+    def test_get_number(self, mock_repo, edge_service):
+        var = FeatureStateBaseHolder({'id': '123', 'key': 'FEATURE_TITLE_TO_UPPERCASE',
```

Comment:
the key name isn't "bla" - you should probably use "bla" as the key name even though this object largely isn't used except for the value. "var" seems like it could be {value: 3} ?